tailwind.config.js

```diff
@@ -2,9 +2,31 @@
 module.exports = {
   content: ['./src/**/*.{html,ts}', 'src/**/*.{css,scss}'],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        accent: '#3545E9',
+        warning: '#FEFD54',
+        success: '#63C995',
+        error: '#E23D69',
+        primary: '#222328',
+        secondary: '#969AB0',
+        bright: '#FFFFFF',
+        muted: '#7C9CBF',
+        shade: '#DBE2EA',
+      },
+      fontFamily: {
+        'ibm-plex-sans': ["IBMPlexSans"]
```

Comment:
do we still need this?

Reply:
I think yes, because this font is for project that I need to set in tailwindcss.config file. 
Or not ?

Reply:
you’ve added the font to styles.scss for the entire project. They should be applied for all elements, so you do not need to provide tailwind rule to specify it it for each element. Do fonts from styles.scss work for your project?

Reply:
Oh, I see, you didn't provide it in the body. Ok, in that case, you'd better provide it in the body tag so you shouldn't do that via tailwind in each element.

Reply:
if i do this `<h1 class="font-[IBMPlexSans] font-semibold text-7xl text-muted">Hello, {{ title }}</h1>`, my font is applied to text. 
Also I will add a font to the body tag style, it will be better that way

Reply:
yes, if your project should have as a default one type of font, it should be applied on the top-level

Reply:
then can I merge branches ?)